### ISSUE TITLE
In the windows explorer in the open dialog file list file names are read twice

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#10620]
### Steps to reproduce:
1. for example, open the NVDA add-ons manager, and press on install add-on button.
2. when pressed, the open dialog appears to chose the add-on.
3. press shift+tab twice to go to the file list.
4. find the desired file you need
### Actual behavior:
file and folder names are pronounced twice, for example ad-on updater.nvda-addon, or the folder name add-ons add-ons
### Expected behavior:
Folder and file names should be spoken once, as in NVDA 2019.2.1
#### NVDA installed/portable/running from source:
installed and portable
#### NVDA version:
latest NVDA beta 2019.3 beta 1 and the last snap from the alpha branch
#### Windows version:
10.0.18363
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
DualCore Intel Core i7-7500U, 3500 MHz (35 x 100),
8GB DDR 4 (2133)
tested also on my lenovo ideapad 320 15isk intel i3 6006U
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2019.2.1, this version pronounces file and folder names in the windows explorer open dialog file  list once.

**Comment by XLTechie** [comment]
Actually, I can replicate that bug in 2019.2.1.

**Comment by XLTechie** [comment]
> 2019.2.1, this version pronounces file and folder names in the windows explorer open dialog file list once.

What about in the add-on installer dialog? I am getting your double-speak 
behavior there in 2019.2.1.

**Comment by lukaszgo1** [comment]
This is a duplicate of #2395 and it is definitely not a regression.

**Comment by zstanecic** [comment]
Can you give arguments, why this is not a bug?

Users see this like a bug, because it’s spurious

**Comment by zstanecic** [comment]
Luke…

It’s normal for portable copies, but not for installed

**Comment by lukaszgo1** [comment]
@zstanecic wrote;

> 
> Can you give arguments, why this is not a bug? Users see this like a bug, because it’s spurious 

Where you've seen me saying that it is not a bug? I only said that it is not a regression because, well duplicate of issue from 2012 cannot be a regression, can it?

**Comment by XLTechie** [comment]
> It’s normal for portable copies, but not for installed

My 2019.2.1 is installed. Win 10 1909.
And that shouldn't be normal for any copies.

**Comment by Brian1Gaff** [comment]
Are tool tips turned on?

One thing I do notice though is that some open dialogues are different to 
others and have different behaviour quite often, some are more like the old 
xp  look and feel, others like the version of windows being run. I've even 
found the sorting order of files different in open dialogues as well, so one 
has to be careful about blaming explorer.

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by zstanecic** [comment]
Hi brian,

Tooltips are off.

**Comment by feerrenrut** [comment]
I can reproduce this issue in 2019.3beta2 and also in 2019.2.1. While it is not a regression it would be good to fix it if we can. It won't get fixed for 2019.3.

**Comment by josephsl** [comment]
Hi, I get a feeling that this is caused by name/description pair where both strings are the same. If so, fixing this should also fix the problem experienced by people trying to install add-ons from Add-ons Manager rather than from File Explorer. Thanks.

**Comment by lukaszgo1** [comment]
@zstanecic Could you please try [this build](https://ci.appveyor.com/api/buildjobs/gvssq2sywrl7cnqi/artifacts/output%2Fnvda_snapshot_pr10750-19620%2C9caea909.exe) and confirm that items are no longer announced twice in places in which they previously were?
